<launch>
    <arg name="model"/>
    <arg name="dataset"/>
    <arg name="epochs" default="1"/>
    <arg name="categories"/>

    <node pkg="rr_ml" type="train_e2e" name="train_e2e" output="screen">
        <param name="epochs" value="$(arg epochs)"/>
        <param name="positive_nonzero_categories" type="str" value="$(arg categories)"/>
        <param name="example_set_dir" value="$(find rr_ml)/data/end_to_end/$(arg dataset)"/>
        <param name="model_output_path" value="$(find rr_ml)/models/end_to_end/$(arg model).h5"/>
    </node>
</launch>
